clients/java/client/src/main/java/org/camunda/bpm/client/task/impl/ExternalTaskImpl.java

```diff
@@ -257,6 +258,9 @@ public <T> T getVariable(String variableName) {
 
     VariableValue variableValue = receivedVariableMap.get(variableName);
     if (variableValue != null) {
+      if(variableValue.getTypedValue() instanceof Executionaware) {

```

Comment:
🔧 Instead of introducing a new interface `ExecutionAware`, can we just check if `instanceof DeferredFileValue`?

Reply:
Good point, thanks!
Incorporated into https://github.com/camunda/camunda-bpm-platform/pull/4594